src/Schema/Events/Conversations.ts

```diff
@@ -14,14 +14,13 @@ import { ActionType } from "."
  *  ```
  *  {
  *    action: "focusedOnConversationMessageInput",
- *    impulse_conversation_id: 198
- *
+ *    impulse_conversation_id: "198"
  *  }
  * ```
  */
 export interface FocusedOnConversationMessageInput {
   action: ActionType.focusedOnConversationMessageInput
-  impulse_conversation_id?: number
+  impulse_conversation_id: string
```

Comment:
Changed this to string to be more generic and required it; in fact, MP returns a string.